blocks/library/audio/index.js

```diff
@@ -80,33 +82,40 @@ registerBlockType( 'core/audio', {
 				}
 				return false;
 			};
-			const controls = (
-				focus && (
-					<BlockControls key="controls">
-						<BlockAlignmentToolbar
-							value={ align }
-							onChange={ updateAlignment }
-						/>
-						<Toolbar>
-							<li>
-								<Button
-									buttonProps={ {
-										className: 'components-icon-button components-toolbar__control',
-										'aria-label': __( 'Edit audio' ),
-									} }
-									type="audio"
-									onClick={ switchToEditing }
-								>
-									<Dashicon icon="edit" />
-								</Button>
-							</li>
-						</Toolbar>
-					</BlockControls>
-				)
+			const controls = focus && (
+				<BlockControls key="controls">
+					<BlockAlignmentToolbar
+						value={ align }
+						onChange={ updateAlignment }
+					/>
+					<Toolbar>
+						<li>
+							<Button
+								buttonProps={ {
+									className: 'components-icon-button components-toolbar__control',
+									'aria-label': __( 'Edit audio' ),
+								} }
+								type="audio"
+								onClick={ switchToEditing }
+							>
+								<Dashicon icon="edit" />
+							</Button>
+						</li>
+					</Toolbar>
+				</BlockControls>
+			);
+
+			const inspectorControls = focus && (
+				<InspectorControls key="inspector">
+					<BlockDescription>
+						<p>{ __( 'Audio, locally hosted, locally sourced.' ) }</p>
```

Comment:
I think you can use an external URL for the audio files.

Reply:
Tweaked the description for the audio block. Good call.